Description: Fix problems in manpages
 In some man page headers, a macro (-*-) is used which makes lintian
  choke and probably should not be there anyway.
Author: Manuel Prinz <manuel@debian.org>
Forwarded: yes
Last-Update: 2011-09-13

Index: openmpi-1.6.5/orte/tools/orterun/orterun.1in
===================================================================
--- openmpi-1.6.5.orig/orte/tools/orterun/orterun.1in	2013-07-01 09:46:41.000000000 +0200
+++ openmpi-1.6.5/orte/tools/orterun/orterun.1in	2013-07-01 09:47:02.000000000 +0200
@@ -1,4 +1,3 @@
-.\" -*- nroff -*-
 .\" Copyright (c) 2009      Cisco Systems, Inc.  All rights reserved.
 .\" Copyright (c) 2008-2009 Sun Microsystems, Inc.  All rights reserved.
 .\" Copyright (c) 2010 Cisco Systems, Inc.  All rights reserved.
